permissions/plugin-depgraph-view.yml

```diff
@@ -4,4 +4,7 @@ github: "jenkinsci/depgraph-view-plugin"
 paths:
 - "org/jenkins-ci/plugins/depgraph-view"
 developers:
-- "wolfs"
\ No newline at end of file
+- "ggrazioli"
+security:
```

Comment:
I don't think this part is needed. The security contact has a special meaning if I remember correctly, so please remove this section.

Reply:
In the docs it tells it just auto-assigns jira issues to the contact (myself), but it's ok for me to drop it.

Reply:
We can setup the auto assign in a different way if you want the issues assigned to you and if @wolfs approves of that.

Reply:
I approve that new issues are automatically assigned to @guidograzioli.

Reply:
Done